integration/maptest/map_test.go

```diff
@@ -15,409 +15,54 @@
-				},
-				{
-					{Index: h2b("0000000000000000000000000000000000000000000000000000000000000000"), LeafValue: []byte("C")},
-				},
-			},
-			get: []struct {
-				revision  int64
-				Index     []byte
-				LeafValue []byte
-			}{
-				{revision: 1, Index: h2b("0000000000000000000000000000000000000000000000000000000000000000"), LeafValue: nil},         // Empty to empty root.
-				{revision: 2, Index: []byte("doesnotexist...................."), LeafValue: nil},                                      // Empty to first root, through empty branch.
-				{revision: 2, Index: h2b("0000000000000000000000000000000000000000000000000000000000000000"), LeafValue: []byte("A")}, // Value to first root.
-				{revision: 3, Index: h2b("0000000000000000000000000000000000000000000000000000000000000000"), LeafValue: []byte("A")},
-				{revision: 4, Index: h2b("0000000000000000000000000000000000000000000000000000000000000000"), LeafValue: []byte("B")},
-				{revision: 5, Index: h2b("0000000000000000000000000000000000000000000000000000000000000000"), LeafValue: []byte("C")},
-			},
-		},
-	} {
-		for _, hashStrategy := range tc.HashStrategy {
-			tree, hasher, err := newTreeWithHasher(ctx, env, hashStrategy)
-			if err != nil {
-				t.Errorf("%v: newTreeWithHasher(%v): %v", tc.desc, hashStrategy, err)
-				continue
-			}
-			pubKey, err := der.UnmarshalPublicKey(tree.GetPublicKey().GetDer())
-			if err != nil {
-				t.Errorf("%v: UnmarshalPublicKey(%v): %v", tc.desc, hashStrategy, err)
-				continue
-			}
-
-			for _, batch := range tc.set {
-				setResp, err := env.MapClient.SetLeaves(ctx, &trillian.SetMapLeavesRequest{
-					MapId:  tree.TreeId,
-					Leaves: batch,
-				})
-				if err != nil {
-					t.Errorf("%v: SetLeaves(): %v", tc.desc, err)
-					continue
-				}
-				t.Logf("Rev: %v Set(): %x", setResp.GetMapRoot().GetMapRevision(), setResp.GetMapRoot().GetRootHash())
-			}
-
-			for _, batch := range tc.get {
-				indexes := [][]byte{batch.Index}
-				getResp, err := env.MapClient.GetLeaves(ctx, &trillian.GetMapLeavesRequest{
-					MapId:    tree.TreeId,
-					Index:    indexes,
-					Revision: batch.revision,
-				})
-				if err != nil {
-					t.Errorf("%v: GetLeaves(): %v", tc.desc, err)
-					continue
-				}
-				t.Logf("Rev: %v Get(): %x", getResp.GetMapRoot().GetMapRevision(), getResp.GetMapRoot().GetRootHash())
-
-				if got, want := getResp.MapLeafInclusion[0].GetLeaf().GetLeafValue(), batch.LeafValue; !bytes.Equal(got, want) {
-					t.Errorf("GetLeaves(rev: %v).LeafValue: %s, want %s", batch.revision, got, want)
-					continue
-				}
 
-				if err := verifyGetMapLeavesResponse(getResp, indexes, batch.revision,
-					pubKey, hasher, tree.TreeId); err != nil {
-					t.Errorf("%v: verifyGetMapLeavesResponse(rev %v): %v", tc.desc, batch.revision, err)
-					continue
-				}
-			}
-		}
-	}
+	os.Exit(m.Run())
 }
 
-func TestInclusion(t *testing.T) {
+func TestMapIntegration(t *testing.T) {
```

Comment:
Perhaps move `TestMapIntegration` to `main_test.go` since it kind of replaces the previous `TestMain` function?

Reply:
Sorry, I don't follow this.